sphinx/themes/basic/static/websupport.js

```diff
@@ -1,808 +0,0 @@
-/*
```

Comment:
I found we still have an asset file in core repository.
We have to move this to sphinxcontrib-websupport package ASAP.

Reply:
done